src/ImageSharp/Colors/Color.cs

```diff
@@ -112,6 +112,9 @@ public Color(uint packed)
             this.packedValue = packed;
         }
 
+        /// <inheritdoc />
+        public BulkPixelOperations<Color> BulkOperations => new BulkPixelOperations<Color>();
```

Comment:
Wouln't it be better to add a comment into `IPixel.BulkOperations` instead, to point the user to `BulkPixelOperations<T>.Instance`?
I seems unlikely to me this will ever cause a trouble, and the current solution is much better in terms of simplicity.